package/MDAnalysis/core/Selection.py

```diff
@@ -1,5 +1,5 @@
 # -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
-# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
```

Comment:
this will result in coverage.py not being able to process this file. i got a patch merged upstream but no new release has been made.